docs/api-docs/channels/channels-quick-start.md

```diff
@@ -0,0 +1,129 @@
+# Building Channels Quick Start
+
+<div class="otp" id="no-index">
+
+
+### On this page
+
+
+- [Create a channel](#create-a-channel)
+- [Create a channel with navigation](#create-a-channel-with-navigation)
+- [Resources](#resources)
+
+</div>
+
+This article is an advanced quick start tutorial for [Bigcommerce technology partners](https://www.bigcommerce.com/partners/) wishing to market their solution within [Channel Manager's](https://developer.bigcommerce.com/api-docs/channels/overview#resources) **Create Channel** flow. For an introduction to channels on BigCommerce, see [Channels Overview](https://developer.bigcommerce.com/api-docs/channels/overview).
+
+
+### Prerequisites

```

Comment:
Currently, the Prerequisites section is not included in On this page. Other tutorials have it called out in On this page. For consistency, I propose to add it to On this page.

Reply:
Its not suppose to be included.